src/webapp/public/locales/de/translation.json

```diff
@@ -23,7 +23,15 @@
           "timer_fade_volume": "Fade volume",
           "toggle_output": "Audio-Ausgang umschalten",
           "sync_rfidcards_all": "Alle Audiodateien und Karteneinträge synchronisieren",
-          "sync_rfidcards_change_on_rfid_scan": "Aktivierung ändern für 'on RFID scan' "
+          "sync_rfidcards_change_on_rfid_scan": "Aktivierung ändern für 'on RFID scan'",
+          "next_song": "Nächster Song",
+          "pause": "Pause",
+          "play": "Abspielen",
+          "prev_song": "Vorheriger Song",
+          "toggle_shuffle": "Shuffle aktivieren/deaktivieren",
+          "toggle_repeat": "Wiederholen aktivieren/deaktivieren",
+          "toggle_repeat_single": "1 wiederholen aktivieren/deaktivieren",
```

Comment:
I know the translation was there before but "1 wiederholen" sound strange. 
Maybe "Song Wiederholen"? 
Or at least capitalize "wiederholen" to sync with `activate-single`?